WcaOnRails/app/models/competition.rb

```diff
@@ -15,18 +15,48 @@ class Competition < ActiveRecord::Base
   has_many :organizers, through: :competition_organizers
   has_many :media, class_name: "CompetitionMedium", foreign_key: "competitionId", dependent: :delete_all
 
+  CLONEABLE_ATTRIBUTES = %w(cityName
+                            countryId
+                            information
+                            eventSpecs
+                            venue
+                            venueAddress
+                            venueDetails
+                            website
+                            latitude
+                            longitude
+                            contact
+                            remarks
+                            use_wca_registration
+                            guests_enabled).freeze
```

Comment:
This looks great! I might have done this to reduce indentation and not make the first/last ones special:

```ruby
CLONEABLE_ATTRIBUTES = %w(
  cityName
  countryId
  information
  eventSpecs
  venue
  venueAddress
  venueDetails
  website
  latitude
  longitude
  contact
  remarks
  use_wca_registration
  guests_enabled
).freeze
```